### DISCUSSION TITLE
[Question][Hw2]How to remove padding when padding is a part of the block

### DISCUSSION THREAD

**Discussion by hong-engi** [prosyslab-classroom/cs348-information-security#691] [Q&A]
Name: 홍은기

We have a block, which is a list of 2 bytes.
When we have padding that has 1 byte size, we cannot remove it in 'dec' directly because we have to return in type of 'block list'. I've tried changing the padding(which is currently 1) to null, but it did not work.
Could someone tell me what method should i use to 'remove' the padding of size 1(byte)? thank you.

**Comment by KAIST-JongchanPark** [comment]
In Ocaml, 0-length list is allowed.
